contracts/root/predicates/ERC20Predicate.sol

```diff
@@ -504,11 +525,11 @@ contract ERC20Predicate is IErcPredicate {
   {
     bytes4 funcSig = BytesLib.toBytes4(BytesLib.slice(txData, 0, 4));
     if (funcSig == WITHDRAW_FUNC_SIG) {
-      require(txData.length == 36, "Invalid tx"); // 4 bytes for funcSig and a single bytes32 parameter
+      // require(txData.length == 36, "Invalid tx"); // 4 bytes for funcSig and a single bytes32 parameter
```

Comment:
why ?

Reply:
This was interesting. So, until now we were assuming say a withdraw tx on matic to be 36 bytes (4 bytes for funcSig and a single bytes32 parameter). 
Now, a `withdraw` in MaticChildErc20 also involves sending ether (msg.value) and tx payload size becomes bigger. So, for this predicate to work for maticEth, I removed this check because it was inconclusive anyway. Do you see an attack vector?